### ISSUE TITLE
SSL error now when playing local mounts directly via port

### ISSUE THREAD

**Issue by nickrobillard** [AzuraCast/AzuraCast#2947]
Ever since updating to the latest Azuracast (I updated 2 days ago), I can no longer play mounts directly via their ports over SSL.

**Why does this matter? Why not just use the built in proxy?**

The reason is so that we can preserve historic URLs from our old system. To do this, I have setup local relay mounts running on our old port (5000) and they have worked until recently.
**Still works**: http://xyz.fm:5000/192
**No longer works**: https://xyz.fm:5000/192

I also noticed that I can no longer play standard mounts via their port either. (I realize there isn't really a reason why one would want to do this, but it is likely related.)
**Works**: https://xyz.fm/radio/8000/hifi
**No longer works**: https://xyz.fm:8000/hifi

For both cases, this is the output that curl gives me:
```
$ curl -v --head https://xyz.fm:5000/192
*   Trying 1xx.xx.xxx.43...
* TCP_NODELAY set
* Connected to xyz.fm (1xx.xx.xxx.43) port 5000 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/cert.pem
  CApath: none
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
* LibreSSL SSL_connect: SSL_ERROR_SYSCALL in connection to xyz.fm:5000 
* Closing connection 0
curl: (35) LibreSSL SSL_connect: SSL_ERROR_SYSCALL in connection to xyz.fm:5000
```

I noticed there are some recent changes around the reverse proxy and I'm guessing this is related. I did follow the instructions here https://github.com/AzuraCast/AzuraCast/issues/2855 (`./docker.sh update-self`, `update` and `letsencrypt-create`) and that all worked well.

**Using Docker installation method**
Yes

**AzuraCast version**
`v0.10.2, #9a956b4 (2020-06-08 0:27) • Docker • PHP 7.4`

**Host Operating System**
`Ubuntu 18.04`

**To Reproduce**

Standard mount case reproduction:
- This is easily reproduced with a standard default Icecast station. If your proxied URL is https://xyz.fm/radio/8000/hifi, request directly to port 8000 with https - https://xyz.fm:8000/hifi - and you will see the issue.

The local relay mount case reproduction steps are as follows:
- Add a `docker-compose.override.yml` and add port 5000 (as directed here https://www.azuracast.com/help/docker/#expand-the-docker-radio-station-port-range)
- Run `docker-compose down` and `docker-compose up -d`
- Turn on "Use Web Proxy for Radio".
- Create an Icecast station on port 8000 with autodj turned on and add a mount point with path `/dubandbass/hifi`.
- Then create another Icecast station on port 5000 with auto dj turned off.
<img width="905" alt="Screen Shot 2020-06-08 at 11 21 39 PM" src="https://user-images.githubusercontent.com/135516/84113112-d8999880-a9de-11ea-9ace-442e3a303cf0.png">

- Add a mount point and set "Relay Stream URL" to the local stream running on 8000. Like this:
<img width="869" alt="Screen Shot 2020-06-08 at 11 04 29 PM" src="https://user-images.githubusercontent.com/135516/84112276-493fb580-a9dd-11ea-976b-70d886e541d7.png">

- Now play the mount directly via port over http - like http://xyz.fm:5000/dubstep/192 - and observe it playing as expected.
- Now play the mount directly via port over https - like https://xyz.fm:5000/dubstep/192 - and observe SSL error. Chrome & Firefox errors:
<img width="343" alt="Screen Shot 2020-06-08 at 11 31 28 PM" src="https://user-images.githubusercontent.com/135516/84114030-7cd00f00-a9e0-11ea-9d9f-6ee7d6a38a22.png">
<img width="786" alt="Screen Shot 2020-06-08 at 11 32 24 PM" src="https://user-images.githubusercontent.com/135516/84114033-7e013c00-a9e0-11ea-8942-56cd3d196754.png">

**Relevant Logs**
Nothing is shown in any logs when the SSL error occurs (none of: `docker logs -f azuracast_stations`, `docker logs -f azuracast_web`, `docker logs -f azuracast_nginx_proxy_1`, `docker logs -f azuracast_nginx_proxy_letsencrypt_1`)

**Comment by nickrobillard** [comment]
Following up here. While this issue persists, it is no longer a problem for me. To preserve old urls, I changed my approach. I found that rather than using relays, I was much better off simply proxying these old urls using Nginx. A big advantage in doing this is that all stats are kept on the new Azuracast station, rather than being split between 2 stations. In case this is useful for anyone else, I documented what I did and added a documentation PR here - https://github.com/AzuraCast/azuracast.com/pull/45
